scantext/admin.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This file is not needed.

Reply:
This file will be discarded int the upcoming commit